monitoring/healthz/src/pages/Nodes.tsx

```diff
@@ -31,31 +31,30 @@ export default function Nodes() {
         <table className="min-w-full divide-y divide-gray-200">
```

Comment:
this is just visual changes we discussed (darker purple bar, left-aligned column headers, simplified repair columns). also sets staging to an artificial limit of 400GB instead of 4TB so now its purple is visible (it was tiny before and made it look like a bug)